csharp/src/Drivers/BigQuery/BigQueryParameters.cs

```diff
@@ -35,6 +35,7 @@ public class BigQueryParameters
         public const string Scopes = "adbc.bigquery.scopes";
         public const string IncludeConstraintsWithGetObjects = "adbc.bigquery.include_constraints_getobjects";
         public const string GetQueryResultsOptionsTimeoutMinutes = "adbc.bigquery.get_query_results_options.timeout";
+        public const string CreateReadSessionMaxStreamCount = "adbc.bigquery.create_read_session.max_stream_count";
```

Comment:
Can we name this something a little more meaningful to the user like `adbc.bigquery.max_fetch_concurrency`?

Reply:
have changed

Reply:
Could you please change the constant name to match the property name?

Reply:
OK, thanks for the correction.